python/ray/data/_internal/logical/operators/all_to_all_operator.py

```diff
@@ -80,6 +77,10 @@ def __init__(
         )
         self._seed = seed
 
+    def aggregate_output_metadata(self) -> BlockMetadata:
```

Comment:
`aggregate_output_metadata` isn't implemented for `AbstractAllToAll` because `Aggregate` changes the metadata. Alternatively, we could add `aggregate_output_metadata` to `AbstractAllToAll` and override the method for `Aggregate` only, but I'd prefer to be more explicit here

Reply:
oh sorry, i got confused by the GH UI and thought this method was under `AbstractAllToAll`, not `RandomShuffle` op. makes sense